exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,5 @@
 const name = " Daniel  ";
+let trimmedName = name.trim()
+let message = "My name is " + trimmedName + " and my name is " + trimmedName.Length + "characters long";
```

Comment:
When concatenating multiple strings, it is usually preferable to use template literals instead of manual string concatenation.

For example:
```
  return `Hello, my name is ${name} and I am ${age} years old`;
```